lib/ansible/modules/network/ordnance/ordnance_config.py

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'core',
+                    'version': '1.0'}
+
+DOCUMENTATION = """
+---
+module: Ordnance_config
+version_added: "2.2??"
+author: "Alexander Turner (alex.turner@ordnance.io)"
+short_description: Manage Ordnance configuration sections
+description:
+  - Ordnance router configurations use a simple block indent file syntax
+    for segmenting configuration into sections.  This module provides
+    an implementation for working with these configuration sections in
+    a deterministic way.
+extends_documentation_fragment: ordnance
+options:
+  lines:
```

Comment:
`lines` vs `commands` sounds confusing.

I would have expected the default to be `commands`